server/data/licenceClient.ts

```diff
@@ -86,11 +86,12 @@ export class LicenceClient {
     licence: Licence = {},
     stage: string = LicenceStage.DEFAULT,
     version: number = 1,
-    varyVersion: number = 0
+    varyVersion: number = 0,
+    standardConditionsVersion: StandardConditionsVersion = undefined
   ): Promise<number> {
     const query = {
-      text: 'insert into licences (booking_id, licence, stage, version, vary_version) values ($1, $2, $3, $4, $5)',
-      values: [bookingId, licence, stage, version, varyVersion],
+      text: 'insert into licences (booking_id, licence, stage, version, vary_version, standard_conditions_version) values ($1, $2, $3, $4, $5, $6)',
```

Comment:
I wasn't thrilled about having to amend some production code to create integration tests. The `utils` resource should use a separate client and not be coupled to the production implementation, in my opinion.

Happy to refactor the utils into its own client as part of this change. It'll just take extra time.

Reply:
👆 This is assuming the utils endpoint isn't a production endpoint - from what I could tell, it wasn't; I'm pretty sure we disable it in production with the env variables. I might be misremembering though 😅